lib/generators/sorcery/templates/initializer.rb

```diff
@@ -114,7 +114,7 @@
   # config.github.key = ""
   # config.github.secret = ""
   # config.github.callback_url = "http://0.0.0.0:3000/oauth/callback?provider=github"
-  # config.github.user_info_mapping = {:email => "name"}
+  # config.github.user_info_mapping = {:email => "login"}
```

Comment:
Are we sure that `login` provides the user's contact email, and doesn't require additional an permission scope? I know that github treats its login email separately from the public profile email, so might be worth looking into further.